ci/common/test-pyclaragenomics.sh

```diff
@@ -32,6 +32,8 @@ rsync -aP rsync://hgdownload-euro.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bed
 rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bigWigToBedGraph $LOCAL_BIN_DIR/ ||
 rsync -aP rsync://hgdownload-euro.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bigWigToBedGraph $LOCAL_BIN_DIR/
 
+pyflakes .
+pydocstyle *.py
```

Comment:
use google convention for pydocstyle e.g. https://github.com/clara-genomics/ClaraGenomicsAnalysis/blob/dev-v0.5.0/pyclaragenomics/style_check#L18